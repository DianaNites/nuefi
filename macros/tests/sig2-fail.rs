#[macros::entry]
fn e_main(handle: (), table: (), extra: (), extra2: ()) {}

fn main() {}
