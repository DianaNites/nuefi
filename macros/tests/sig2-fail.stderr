error: Unexpected argument
 --> tests/sig2-fail.rs:2:34
  |
2 | fn e_main(handle: (), table: (), extra: (), extra2: ()) {}
  |                                  ^^^^^^^^^

error: Unexpected argument
 --> tests/sig2-fail.rs:2:45
  |
2 | fn e_main(handle: (), table: (), extra: (), extra2: ()) {}
  |                                             ^^^^^^^^^^
